ios/RNDocumentPicker/RNDocumentPicker.m

```diff
@@ -24,9 +24,11 @@ @interface RNDocumentPicker () <UIDocumentPickerDelegate>
 @end
 
 @implementation RNDocumentPicker {
+    UIDocumentPickerMode mode;
+    NSString *copyDestination;
     NSMutableArray *composeResolvers;
     NSMutableArray *composeRejecters;
-    NSString* copyDestination;
+    NSMutableArray *urls;
```

Comment:
question: maybe I'm missing something; where is this array allocated?

Reply:
My bad, it's fixed now.